sw/device/mask_rom/mask_rom.h

```diff
@@ -0,0 +1,56 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#ifndef OPENTITAN_SW_DEVICE_MASK_ROM_MASK_ROM_H_
+#define OPENTITAN_SW_DEVICE_MASK_ROM_MASK_ROM_H_
+
+#include <stdnoreturn.h>
+
+/**
+ * Denotes functions that have to use the interrupt handler ABI.
+ *
+ * These must be 4-byte aligned, and they use a different calling convention.
+ */
+#define MASK_ROM_INTERRUPT_HANDLER_ABI __attribute__((aligned(4), interrupt))
+
+/**
+ * Denotes functions that have to be near the interrupt vector, because they
+ * are jumped to from it.
+ */
+#define MASK_ROM_VECTOR_FUNCTION __attribute((section(".crt")))
+
+/**
+ * The first assembly procedure executed by the Mask ROM (defined in
+ * `mask_rom.S`).
+ *
+ * This procedure does not obey the standard RISC-V calling convention.
```

Comment:
Maybe this suggests we should say something like "Do not all this function from C."?